rs/protobuf/def/types/v1/idkg.proto

```diff
@@ -233,6 +234,13 @@ message SchnorrSigShare {
   bytes sig_share_raw = 3;
 }
 
+message VetKdShare {
+  NodeId signer_id = 1;
+  RequestId request_id = 2;
+  bytes encrypted_key_share = 3;
+  bytes node_signature = 4;
```

Comment:
General question: Are these messages persisted somewhere?

I'm asking because I'm wondering how involved it would be to change the definition of this message. If they are not persisted anywhere, then changing this should be painless in the sense that we don't have to think about backward-compatibility, right?

Background: if we were ever to switch away from the "optimistic" variant of the underlying cryptographic vetKD protocol, then we don't need the `node_signature` any more, but just the `encrypted_key_share`.

(We could theoretically completely hide the fact that from the viewpoint of the Crypto component an "encrypted vetKD key share" is actually (1) the encrypted_key_share itself and (2) a signature of the node on the encrypted_key_share by serializing the entire `ic_types::crypto::vetkd::VetKdEncryptedKeyShare` into the `encrypted_key_share`, but that doesn't seem like a clean solution.)

Reply:
They are persisted, but not across upgrades. That means we don't have to worry about compatibility when changing these types

Reply:
I see, thanks!